### DISCUSSION TITLE
Old Capture Card

### DISCUSSION THREAD

**Discussion by nerdgh0st** [ntsc-rs/ntsc-rs#106] [Presets]
This preset emulates a capture card recording (reminiscent of old YouTube Xbox 360/Wii/PS3 recordings from 2006-2013)
[ccardnewest.json](https://github.com/valadaptive/ntsc-rs/files/15444099/ccardnewest.json)
![ntsc-rs-standalone_c3JxyHx2lN](https://github.com/valadaptive/ntsc-rs/assets/68407522/4fed1ebe-5186-4b1f-ae5d-858bb3c7c5c4)
![ntsc-rs-standalone_WsHyZzE9Et](https://github.com/valadaptive/ntsc-rs/assets/68407522/b62cdd00-030e-4357-b33d-505a6a7801d9)
![ntsc-rs-standalone_WgeRcdSz89](https://github.com/valadaptive/ntsc-rs/assets/68407522/5a972c5d-694c-40c9-964e-6c867868d20d)

**Comment by smfthe1199** [comment]
Ooh that's great! Thanks for this!

**Reply by smfthe1199** [reply]
@nerdgh0st can I do some modifications to this as well?

**Reply by nerdgh0st** [reply]
Yes you can.... sorry for the late reply though

**Comment by Degamisu** [comment]
> This preset emulates a capture card recording (reminiscent of old YouTube Xbox 360/Wii/PS3 recordings from 2006-2013)

Honestly, i feel like this is any sort of capture card. happens to mine too! excellent work btw.

**Comment by Maincrosift** [comment]
hey, that's pretty good!

**Comment by j4wassup** [comment]
thank you
https://www.youtube.com/watch?v=flC589U-xqk&feature=youtu.be
